docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+concourse-db:
```

Comment:
We can. Please take a look at my comment on the pivotal story and give me final shout.